uk.co.compucorp.civicrm.hrcore/tests/phpunit/CRM/HRCore/Hook/BuildForm/ContactAdvancedSearchTest.php

```diff
@@ -2,6 +2,11 @@
 use CRM_HRCore_Test_BaseHeadlessTest as BaseHeadlessTest;
 use CRM_HRCore_Hook_BuildForm_ContactAdvancedSearch as ContactAdvancedSearch;
 
+/**
+ * Class CRM_HRCore_Hook_BuildForm_ContactAdvancedSearchTest
```

Comment:
I don't think we usually have this `Class` docblock

Reply:
From a quick look, it seems that most of the test classes actually do have that. It's added automatically by PhpStorm and since we don't really have a rule about it I'd prefer to keep this one for now, until we decide if we should have it or not and them apply that in all places